catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/suggestion/UtmUpsCoordinateProcessor.java

```diff
@@ -76,42 +79,178 @@ public UtmUpsCoordinateProcessor(CoordinateSystemTranslator translator) {
+          .collect(toList());
     }
+  }
+
+  private LiteralSuggestion suggestion(UtmUpsCoordinate utmUps) {
     return new LiteralSuggestion(
         LITERAL_SUGGESTION_ID,
-        nameBuilder.toString(),
+        makeSuggestionText(utmUps),
+        Collections.singletonList(latLonFromUtmUtps(utmUps)));
+  }
+
+  /**
+   * Generates one {@link LiteralSuggestion} by combining the latitude/longitude values of each
+   * UTM/UPS coordinate in the {@code utmUpsCoords} list. This method is used to generate a
+   * suggestion when the user enters multiple UTM/UPS coordinates. No additional suggestions are
+   * made for each point for simplicity since the map will pan to the extent or bounding box of all
+   * the points. As a result, the N/S characters of each coordinate are always treated as latitude
+   * bands and a coordinate is considered invalid if a latitude band is not included. Example: "12S
+   * 241451mE 4101052mN 13R 37090mE 63394439mN" -> "UTM/UPS: [ 12S 241451mE 4101052mN ] [ 13R
+   * 37090mE 63394439mN ]"
+   *
+   * @param utmUpsCoords a list of strings matching the {@code PATTERN_UTM_OR_UPS_COORDINATE}
+   *     pattern, created from the user-provided gazetteer query text.
+   * @return a singleton list with one {@link LiteralSuggestion} with the combined list of
+   *     latitude/longitude values.
+   */
+  private LiteralSuggestion suggestionForMultiplePoints(final List<String> utmUpsCoords) {
+    List<UtmUpsCoordinate> utmUpsList =
         utmUpsCoords
             .stream()
-            .map(this::toLatLon)
+            .map(this::parseUtmUpsString)
             .filter(Objects::nonNull)
-            .map(d -> new LatLon(d.getLat(), d.getLon()))
-            .collect(Collectors.toList()));
+            .collect(toList());
+    List<LatLon> latLonList =
+        utmUpsList.stream().map(this::latLonFromUtmUtps).filter(Objects::nonNull).collect(toList());
+    String suggestionText = makeSuggestionText(utmUpsList);
+
+    return new LiteralSuggestion(LITERAL_SUGGESTION_ID, suggestionText, latLonList);
+  }
+
+  private static String makeSuggestionText(UtmUpsCoordinate utmUps) {
+    final StringBuilder nameBuilder = new StringBuilder();
+    if (utmUps.isUTM()) {
+      nameBuilder.append("UTM: [ ").append(utmUps.toString()).append(" ]");
+      if (utmUps.getLatitudeBand() == null) {
+        if (utmUps.getNSIndicator() == NSIndicator.SOUTH) {
+          nameBuilder.append(" (Southern)");
+        } else {
+          nameBuilder.append(" (Northern)");
+        }
+      }
+    } else {
+      nameBuilder.append("UPS: [ ").append(utmUps.toString()).append(" ]");
+    }
+    return nameBuilder.toString();
+  }
+
+  private static String makeSuggestionText(List<UtmUpsCoordinate> utmUpsList) {
+    final StringBuilder nameBuilder = new StringBuilder();
+    long numberOfUtmCoords = utmUpsList.stream().filter(UtmUpsCoordinate::isUTM).count();
+    int total = utmUpsList.size();
+    if (numberOfUtmCoords == total) {
```

Comment:
❗️ This is not going to accomplish exactly what you want. The `isUTM` and `isUPS` functions tell you if the coordinate "data" exists in the "space", not what the text format is. On top of that, there is overlap, so both functions could return true. They are not mutually exclusive. 

See my comments on my review of the PR against master: https://github.com/codice/ddf/pull/5718#pullrequestreview-342925894

Reply:
✅ After further analysis we could not reproduce the problem. Turns out the semantics of `UtmUpsCoordinate` are different from the other coordinate impls. This is fine.